samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/AppCatalog.cs

```diff
@@ -0,0 +1,394 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using AdaptiveCards;
+
+namespace AppCatalogSample.Helper
+{
+  public class AppCatalog
+    {
+        public static  string gToken { get; set; }
+        public string GetToken(string token)
+        {
+            gToken = token;
+            return token;
+        }
+
+        public static async Task<IList<TeamsApp>> GetAllapp()
```

Comment:
This method is causing compilation error as Not all the paths return values. Could you please fix this?